pkg/controller/datavolume-controller.go

```diff
@@ -637,12 +637,16 @@ func (r *DatavolumeReconciler) getCloneStrategy() (cdiv1.CDICloneStrategy, error
 		return cdiv1.CloneStrategySnapshot, fmt.Errorf("no active CDI")
 	}
 
-	if cr.Spec.CloneStrategyOverride == nil {
+	if cr.Spec.Config == nil {
```

Comment:
Nitpick: you can do:
```go
if cr.Spec.Config == nil || cr.Spec.Config.CloneStrategyOverride == nil {
  return cdiv1.CloneStrategySnapshot, nil
}
Instead of the two ifs. No functional difference really but groups the two together
```

Reply:
Done